src/components/editor/editor-pane/tool-bar/table-picker/table-picker.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+
+
+.table-picker-container {
```

Comment:
All rules in the root level of this file are available for other components. So please namespace the other rules under this one.

Reply:
(you can also reduce the number of light imports using the namespace)

Reply:
done